disnake/guild.py

```diff
@@ -1935,6 +1936,74 @@ async def create_scheduled_event(
         data = await self._state.http.create_guild_scheduled_event(self.id, reason=reason, **fields)
         return GuildScheduledEvent(state=self._state, data=data)
 
+    async def fetch_welcome_screen(self):
```

Comment:
Return type is missing

Reply:
https://github.com/DisnakeDev/disnake/pull/339/commits/83b9bb72675cb860ca6c030babcf0c12ef808d25